astropy/utils/xml/writer.py

```diff
@@ -148,6 +153,20 @@ def start(self, tag, attrib={}, **extra):
         return len(self._tags)
 
     @contextlib.contextmanager
+    def xml_escaping(self, enabled=True, clean_kwargs=None):
+        current_xml_escape_cdata = self.xml_escape_cdata
+        if not enabled:
+            if HAS_BLEACH:
+                if clean_kwargs is None:
+                    clean_kwargs = {}
+                self.xml_escape_cdata = lambda x: bleach.clean(x, **clean_kwargs)
+            else:
+                raise ValueError('bleach package is required when HTML escaping is disabled.\n'
+                                 'Use "pip install bleach".')
```

Comment:
I find this kind of confusing...  I think it would make more sense if `xml_escaping` just always requires bleach and raises an `ImportError` if this context manager is used without it installed.  Then in the HTML reader's `__init__` immediately raise an exception if `raw_html_cols` is given without `bleach` installed.  That way it becomes immediately obvious that that specific option cannot be used, rather than failing with an exception during writing (which doesn't make it as immediately clear that this is due to the use of `raw_html_cols`).